src/bindings/swig/CMakeLists.txt

```diff
@@ -6,6 +6,11 @@ if (NOT SWIG_FOUND)
 	return ()
 endif ()
 
+if (ENABLE_ASAN)
+	exclude_binding (swig "SWIG is not compatible with ENABLE_ASAN")
+	return ()
```

Comment:
Would this return be problematic once we change the bindings to behave like the plugins?

Reply:
I don't expect so